src/main/kotlin/fi/metatavu/metaform/server/rest/RepliesApi.kt

```diff
@@ -237,6 +240,7 @@ class RepliesApi: fi.metatavu.metaform.api.spec.RepliesApi, AbstractApi() {
     auditLogEntryController.generateAuditLog(metaform, userId, reply.id!!, null, null, AuditLogEntryType.DELETE_REPLY)
     replyController.deleteReply(reply)
 
+
```

Comment:
Move ReplyEvent triggering to here from the controller.

Reply:
As per discussed, did this change in order to keep the logic intact.